tests/unit/modules/boto_vpc_test.py

```diff
@@ -436,14 +442,20 @@ def test_that_when_describing_vpc_by_id_it_returns_the_dict_of_properties_return
         '''
         Tests describing parameters via vpc id if vpc exist
         '''
+        # With moto 0.4.25 is_default is set to True. 0.4.24 and older, is_default is False
+        if _get_moto_version() >= LooseVersion('0.4.25'):
+            is_default = True
+        else:
+            is_default = False
+
         vpc = self._create_vpc(name='test', tags={'test': 'testvalue'})
 
         describe_vpc = boto_vpc.describe(vpc_id=vpc.id, **conn_parameters)
 
         vpc_properties = dict(cidr_block=unicode(cidr_block),
-                              is_default=False,
+                              is_default=is_default,
                               state=u'available',
-                              tags={'Name': 'test', 'test': 'testvalue'},
+                              tags={u'Name': u'test', u'test': u'testvalue'},
```

Comment:
Are you positive that these should be designated as Unicode strings as a return from this function?